src/js/Lotto.js

```diff
@@ -0,0 +1,47 @@
+import { WARNING_WHEN_NOT_IN_1000_UNITS, UNIT_PRICE } from "./utils/consts.js";
+
+class Lotto {
+  #price
+  constructor(price) {
+    this.#price = price
+  }
+
+  isPriceValid() {
+    if (this.#price < UNIT_PRICE || this.#price >= (UNIT_PRICE * 1000)) return false
+
+    if ((this.#price % UNIT_PRICE) !== 0) return this.valueInitAndAlertNotUnitOfthousand()
+
+    return true
+  }
+  
+  valueInitAndAlertNotUnitOfthousand() {
+    alert(WARNING_WHEN_NOT_IN_1000_UNITS)
+    return false;
+  }
+
+  get lottoTicketAmount() {
+    return this.isPriceValid() ? (this.#price / UNIT_PRICE) : null
+  }
+
+  static createRandomNumberFromOneToFortyFive(count) {
+    const lottoRandomNumbers = []
+    // 1 ~ 45 숫자로 구성된 배열 만들기
+    // 랜덤으로 셔플 하는 함수 가져오기
+    // 6개로 잘라서 가져옴
```

Comment:
```suggestion
    // 1 ~ 45 숫자로 구성된 배열 만들기
    const numbers = Array(45).fill(1).map((v, k) => v + 1);
    
    // 랜덤 셔플 + 6개 자르기
    const lottoNumbers = numbers.slice().sort(() => Math.random() - 0.5).slice(0, 6);
```
이 부분은 이렇게 단 두 줄로 표현할 수 있답니다 ㅎㅎ
배열과 관련된 함수를 잘 알고 있어야 한다는 느낌이 들지않나요!?

Reply:
다시 중요한점은,
lotto가 지금 너무 많은 역할을 하고 있는 것 같아요
저는 넉넉하게 5개의 객체로 분리할 수 있다고 생각합니다!